docs/general/getting_started.md

```diff
@@ -80,6 +80,55 @@ end
 
 ActiveModelSerializers will use `PostSerializer::CommentSerializer` (thus including only the `:body_short` attribute) when serializing a `Comment` as part of a `Post`, but use `::CommentSerializer` when serializing a `Comment` directly (thus including `:body, :date, :nb_likes`).
 
+### Polymorphic Relationships
+
+As of release `0.10`, polymorphic relationships are serialized just by specifying the relationship. For example:
+
+```ruby
+class PictureSerializer < ActiveModel::Serializer
+  has_one :imageable
+end
+```
+
+If the instance of `Picture` belongs to a `Product`, an example payload might be:
+
+```javascript
+{
+  picture: {
+    id: 1,
+    imageable: {
+      id: 3,
+      title: "Product 3"
+    }
+  }
+}
+```
+
+In prior releases (and optionally in `0.10`), a `polymorphic: true` parameter is required:
+
+```ruby
+class PictureSerializer < ActiveModel::Serializer
+  has_one :imageable, polymorphic: true
+end
+```
+
+In this case, an example payload might be:
+
+```javascript
+{
+  picture: {
+    id: 1,
+    imageable: {
+      type: "product",
+      product: {
+        id: 3,
+        title: "Product 3"
+      }
+    }
+  }
+}
+```
+
```

Comment:
:100: Should this doc change be in this file?  Maybe in `serializers` or `layouts and rendering`? Just a thought, this might be the best spot, as well.

Is there a test for this behavior? Would be great to add.